### PR TITLE
Issue 191 holdovers

### PR DESCRIPTION
**Pull request by oodler577**
Bringing over files, one per commit in my #191 branch; perl https://gist.github.com/jkeenan/8507fb2790dee4db60ef95406ed3c584

### CODE REVIEW

**Comment by oodler577** [comment]
redoing

File: t/op/svleak.t [hunk 1]
```diff
@@ -511,7 +510,6 @@ leak(5, 0, sub { scalar &const_av_xsub_leaked }, "const_av_sub in scalar context
 # check that OP_MULTIDEREF doesn't leak when compiled and then freed
 
 eleak(2, 0, <<'EOF', 'OP_MULTIDEREF');
-no strict;
```
Comment (atoomic) [inline t/op/svleak.t:514]: we should probably leave that `no strict`
it's already like this in blead, and we should not remove it IMO

**Review by atoomic — CHANGES_REQUESTED** [review]
one minor issue in `t/op/svleak.t` otherwise this looks pretty good

File: t/op/svleak.t (hunk 1 above)
Comment (jkeenan) [reply t/op/svleak.t:514]: > we should probably leave that `no strict`
> it's already like this in blead, and we should not remove it IMO

What if it was never really needed in blead?

Comment (atoomic) [reply t/op/svleak.t:514]: then my advice would be let's fix it in blead and not there

**Comment by jkeenan** [comment]
Re-closing, as @atoomic is handling individual pull requests.

**Comment by atoomic** [comment]
I was fine with both, individual or mass update :-)
the individual one would allow a better bisect if needed

**Pull request closed without merging**